Firestore/core/src/core/key_field_not_in_filter.cc

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "Firestore/core/src/core/key_field_not_in_filter.h"
+#include "Firestore/core/src/core/key_field_in_filter.h"
+
+#include <memory>
+#include <utility>
+
+#include "Firestore/core/src/model/document.h"
+#include "Firestore/core/src/model/document_key.h"
+#include "absl/algorithm/container.h"
+
+namespace firebase {
+namespace firestore {
+namespace core {
+
+using model::Document;
+using model::DocumentKey;
+using model::FieldPath;
+using model::FieldValue;
+
+using Operator = Filter::Operator;
```

Comment:
It seems like `query_core.cc` could benefit quite a bit from applying this using statement there (it would allow you to shorten all the `Filter::Operator::Foo` to just `Operator::Foo`). Here we're only accessing the shortened `Operator` just the once. This inconsistency seems glaring.